### ISSUE TITLE
Why use big weight decay?

### ISSUE THREAD

**Issue by volgachen** [facebookresearch/deit#68]
I found that weight decay is set to 0.05 in your code. It is much larger than common value in ConvNets (like 1e-4). I wonder how it will affect the results. Could you share the results using other values if you have tested?

I also wonder why the learning rate is much smaller than ConvNets.

**Comment by TouvronHugo** [comment]
Hi @volgachen ,
The difference with CNN for the learning rate and weigth decay is due to the fact that we use AdamW not SGD.
Best,
Hugo

**Comment by volgachen** [comment]
> Hi @volgachen ,
> The difference with CNN for the learning rate and weigth decay is due to the fact that we use AdamW not SGD.
> Best,
> Hugo

Thank you for your reply. Have you tried SGD? How about its performance?

**Comment by TouvronHugo** [comment]
Yes but the performance are not very good (74.5% with the Base model)
Best,
Hugo

**Comment by volgachen** [comment]
Thank you for sharing!
